content/02.introduction.md

```diff
@@ -78,29 +74,49 @@ The symptoms of infection by SARS-CoV-2 can vary greatly, making it difficult fo
+The development of diagnostic tools is critical to this goal.
+Accurate diagnoses on a large scale is necessary for collected the data needed to develop epidemiological models.
+Other areas of public health that address resource availability, inequity, human behavior, and other components that influence people's exposure to pathogens and ability to manage illness will also be critical to mounting a global response to the pandemic.
+Currently, this manuscript focuses primarily on contextualizing epidemiological characteristics such as reproduction number and dynamics of transmission that intersect with the fundamental biology of the virus or the development of therapeutic and diagnostic technologies.
+
+#### 2.5.c Biotechnology
+
+##### 2.5.c.i Diagnostics
 
 Two major concerns within diagnosis include the detection of current infections in individuals with and without symptoms, and the detection of past exposure without an active infection.
 In the latter category, identifying whether individuals can develop or have developed sustained immunity is also a major consideration.
+The development of high-throughput, affordable methods for detecting active infections and sustained immunity will be critical to understanding and controlling the disease.
+
+- What are approaches that allow us to detect current infection or past exposure for other viruses?
+- What is sustained immunity and what are the indicators?
+
+##### 2.5.c.ii Therapeutics
 
-Within therapeutics, some possible efforts include efforts to identify strategies for the management of symptoms as well as the development of antivirals...
+The identification of interventions that can mitigate the effect of the virus on exposed and infected individuals is a significant research priority.
+Some possible approaches include the identification of existing pharmaceuticals that reduce the severity of infection, either by reducing the virus' virulence (e.g., antivirals) or managing the most severe symptoms of infection.
+Due to the long timeline for the development of novel pharmaceuticals, in most cases, research surrounding possible pharmaceutical interventions focuses on the identification and investigation of existing compounds whose mechanisms may be relevant to COVID-19.
+Other foci of current research include the identification of antibodies produced by survivors of COVID-19 and the development of vaccines.
+Understanding the mechanisms describing host-virus interactions between humans and SARS-CoV-2 are thus critical to identifying candidate therapeutics.
 
+### 2.6 Summary
 In this review, we seek to consolidate information about efforts to develop strategies for diagnosis and therapeutics as new information is released by the scientific community.
+We include information from both traditional peer-reviewed scientific literature and from preprints, which typically have not undergone peer review.
```

Comment:
Maybe make a comment along the line of " but have been critically appraised". It might not be necessary though if you have mentioned this in the methodology sections.

Reply:
Like this?
```suggestion
We include information from both traditional peer-reviewed scientific literature and from preprints, which typically have not undergone peer review but have been critically appraised by the scientists involved in this effort.
```

Reply:
Perfect, exactly